### ISSUE TITLE
[Bug]: installed mods keep uninstalling

### ISSUE THREAD

**Issue by Zanthiem** [FTBTeam/FTB-App#740]
### What Operating System

Windows 10

### Debug Code

FTB-DBGEXALOVOKUQ

### Describe the bug

the last 2 times i have opened the app my installed mods are gone and save files are lost

### Steps to reproduce

close app
shut down pc
get on to play next day
all installed mods are gone

### Expected behaviour

log in and be able to play stoneblock 3 and not restart for the 4th time

### Information

- [X] I have provided as much information as possible

**Comment by ViktorKorladinov** [comment]
Having the same problem, launcher is unusable, all my saves are once again gone.

**Comment by MichaelHillcox** [comment]
Do you mean modpacks?

**Comment by ViktorKorladinov** [comment]
Yes, happened with both stoneblock 3 and oceanblock, so I doubt the issue
is with the modpack - I think the issue occurs when the game crashes and
then the launcher is closed. When I run and stop the game  normally after a
crash, next time I open the launcher the modpack is still there. Moreover,
I checked the launcher files and it doesn't seem to delete the modpacks. -
it fails to recognize them, I think.

**Comment by DisMadMan** [comment]
Same just happened to me. I thought it was a one and done kind of thing, but that doesn't seem to be the case. I spent all day yesterday working on my Stoneblock 3 save, and lost 10 hours of progress.

**Comment by DisMadMan** [comment]
Okay. I noticed this was a duplicate issue, so I tried one of the suggested fixes and that was to redownload the instance, delete all of its files. and copy and paste the old ones in their place. It does work.
